includes/phing/src/Tasks/PlatformVersionsTask.php

```diff
@@ -76,11 +76,11 @@ public function main()
         );
 
         // Get latest version of Platform.
-        $resp=$this->callGithubReleases('https://api.github.com/repos/ec-europa/platform-dev/releases/latest');
+        $resp = $this->callGithubReleases('https://api.github.com/repos/ec-europa/platform-dev/releases/latest');
```

Comment:
If `$latest_version` match  `$this->_packageVersion` this mean user specified the full version into prop, we can skip all logic and return it now. No need inspect `releases` endpoint.

Reply:
Implemented.